eq-author/src/App/publish/PublishPage.js

```diff
@@ -27,6 +27,26 @@ export const themes = [
   "Social",
 ];
 
+export const ErrorContext = styled.div`
+  position: relative;
+
+  ${props =>
+    props.isInvalid &&
+    css`
```

Comment:
Never knew you could do that 'css' keyword bit. Definitely looks tidier than what I've tried to do in the past. Nice, I'll be thiefing that in future. ;)